### PR TITLE
Change Ontology IRI

### PR DESCRIPTION
**Pull request by nck-mlcnv**
Also does:
- removes unused test files
- add file upload of the ontology to the ftp server

### CODE REVIEW

**Review by bigerl — CHANGES_REQUESTED** [review]
Thx for the PR. This seems not fully completed yet. 

1. Is the distribution at that path clarified? A action that publishes the file on release would be good. 
2.Three are still quite some occurrences of `iguana-benchmark.eu` left:
image

**Comment by nck-mlcnv** [comment]
The test files are all not used actually, I will remove them soon

**Comment by bigerl** [comment]
I think the files should then be removed in this PR so that we don't have to change them.

File: .github/workflows/deployment_test.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
         uses: SamKirkland/FTP-Deploy-Action@v4.3.5
         with:
           server: files.dice-research.org
```
Comment (bigerl) [inline .github/workflows/deployment_test.yml:53]: it should be ftp over tls. Not sure how to set that up here

Comment (nck-mlcnv) [reply .github/workflows/deployment_test.yml:53]: Yeah I thought so too, but something didn't work on the first run

Comment (bigerl) [reply .github/workflows/deployment_test.yml:53]: maybe `ftps-legacy`?

Comment (nck-mlcnv) [reply .github/workflows/deployment_test.yml:53]: didn't work either

Comment (nck-mlcnv) [reply .github/workflows/deployment_test.yml:53]: I don't know if this is relevant but it seems to be a common [issue](https://github.com/SamKirkland/FTP-Deploy-Action/issues/106) . I think I will just try to use lftp itself.

**Review by bigerl — APPROVED** [review]
Changes look now good to me.

**Pull request merged**